x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/event_merging_banner.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiCallOut, EuiLink, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { useKibana } from '@kbn/kibana-react-plugin/public';
+
+export const EventMergingBanner: React.FC<{ onDismiss: () => void }> = ({ onDismiss }) => {
+  const { docLinks } = useKibana().services;
+  const bannerTitle = i18n.translate(
+    'xpack.securitySolution.endpoint.policy.eventMergingBanner.title',
+    {
+      defaultMessage: "We've recently made changes to Linux event collection",
+    }
+  );
+
+  return (
+    <EuiCallOut title={bannerTitle} onDismiss={onDismiss} data-test-subj="eventMergingCallout">
+      <EuiText size="s">
+        <FormattedMessage
+          id="xpack.securitySolution.endpoint.policy.eventMergingBanner.body"
+          defaultMessage="Elastic Agent 8.16+ will produce less telemetry without reducing system visibility, which may impact existing event filters. To learn more about these changes and how to adjust your settings, visit our "
+        />
+        <EuiLink
+          href={docLinks?.links.securitySolution.eventMerging}
+          target="_blank"
+          data-test-subj="eventMergingDocLink"
+        >
+          <FormattedMessage
+            id="xpack.securitySolution.endpoint.eventMergingBanner.doc.link"
+            defaultMessage="documentation."
+          />
+        </EuiLink>
+      </EuiText>
+    </EuiCallOut>
+  );
```

Comment:
To be consistent between different banners, I would put the link icon before the dot, just like in the banner below.

![Screenshot 2024-10-09 at 12 09 05](https://github.com/user-attachments/assets/25d39205-e0fc-4878-bc9c-d0a1988f51ce)

Example of the structure to achieve this - https://github.com/elastic/kibana/blob/76b8fae8810b2f3dc3b623effb35beab8c950460/x-pack/plugins/security_solution/public/management/pages/policy/view/policy_settings_form/components/related_detection_rules_callout.tsx#L25

Reply:
🦅 👁️ ! :)
5f7ca44e516d0b7c17f424c386c396b90c0af0ae